tests/email.service.test.ts

```diff
@@ -1,70 +1,106 @@
 import { sendEmailWithTemplate } from '../src/lib/email.service';
-import { ITemplateRequest } from '../src/lib/goodModels/ITemplateRequest';
+import { ITemplateRequest } from '../src/lib/models/ITemplateRequest';
+import { setMandrillApiKey } from '../src/config/client-config';
+
+const testEmailAddress = 'ben.vanraemdonck@icapps.com';
 
 describe('email service', () => {
   describe('sendEmailWithTemplate', () => {
-    it('Should send the email correctly with values filled in for recipient', async () => {
+    beforeAll(() => {
+      setMandrillApiKey('YY5pkBrTRAXKodN-0CL38g');
+    });
 
-      /*
-            // SCENARIO 1
-          const values = {
-              subject: 'My subject',
-              from: 'info@icapps.com',
-              to: 'brent@test.be',
-              variables: [ // global
+    it('Should send the email correctly with values filled in for recipient', async () => {
+      const mailInfo : ITemplateRequest = {
+        templateName: '00-forgot-password',
+        subject: 'My subject',
+        from: { email: 'info@icapps.be', name: 'Info icapps' },
+        to: [{ email: testEmailAddress, name: 'name', content: [{ name: 'firstname', value: 'Custom for recipient' }] }],
+        globalContent: [
                 { name: 'var1', value: 'myValue1' },
                 { name: 'var2', value: 'myValue2' },
-              ],
-              ...allOtherOptions,
-            }; */
+        ],
+      };
 
-      /*
-      /*
-            // SCENARIO 3
-          const values = {
-              subject: 'My subject',
-              from: { email: 'info@icapps.com', name: 'Info icapps }',
-              to: [{ email: 'brent@test.be', name: 'Brent', type: 'cc' }],
-              variables: [ // global
+      await sendEmailWithTemplate(mailInfo);
```

Comment:
Make sure to use jest.spyOn on mandrill. And check with `hasBeenCalledWith`. This way we don't actually call mandrill